IFC4x3/PropertyConstants/r/RUNNINGTRACKFORMULTIPLEUNIT_3ET0GAEm544Po8pbZAXLpv.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RUNNINGTRACKFORMULTIPLEUNIT_3ET0GAEm544Po8pbZAXLpv" Name="RUNNINGTRACKFORMULTIPLEUNIT" UniqueId="ce74040a-3b01-4411-9c88-ce58ca855cf9">
```

Comment:
Propose renaming to `MULTIPLEUNITRUNNINGTRACK`.

Reply:
Updated: https://github.com/bSI-InfraRoom/IFC-Specification/pull/178/commits/028e8e83605d724326e1c6af7cf041c587b9c877